walletlibrary/src/main/java/com/microsoft/walletlibrary/identifier/HolderIdentifierCreator.kt

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved
+
+package com.microsoft.walletlibrary.identifier
+
+import com.microsoft.walletlibrary.did.sdk.crypto.CryptoOperations
+import com.microsoft.walletlibrary.did.sdk.crypto.KeyGenAlgorithm
+import com.microsoft.walletlibrary.did.sdk.crypto.keyStore.EncryptedKeyStore
+import com.microsoft.walletlibrary.did.sdk.crypto.keyStore.toPrivateJwk
+import com.microsoft.walletlibrary.did.sdk.util.controlflow.KeyStoreException
+import com.nimbusds.jose.jwk.JWK
+import com.nimbusds.jose.jwk.KeyUse
+
+internal class HolderIdentifierCreator(private val encryptedKeyStore: EncryptedKeyStore) {
```

Comment:
I see there are comments on private method, but will you add comments to class to internal methods?